### PR TITLE
Window popup

### PR DESCRIPTION
**Pull request by gisachris**
### IN This PR I have done the following 👨‍💻 :
-Refactor project section to load dynamically with javascript file (app.js) 🎡 
-Add pop up modals for there corresponding projects 🥇  
-add some styling to better fit figma design 🎨  and overall website functionality 🧑‍🔧 

-implement the following actions 💥 :
**When the user clicks (or taps) the button to check project details, the popup with details about the project appears.✅**
**When the user clicks (or taps) the close (X) button, the popup disappears. ✅**

I am looking forward to your comments on the work I have done so far 🌟 . thankyou 🙇

### CODE REVIEW

File: styling.css [hunk 1]
```diff
@@ -837,6 +854,194 @@
...
+  }
+
+  .modalbutton2 > a {
+    width: fit-content;
+    color: white;
+    font-family: 'Inter', sans-serif;
+    display: flex;
+    column-gap: 10px;
+    justify-content: center;
+  }
```
Comment (Qoosim) [inline styling.css]: - [ ] Great job so far. However, there're few things to work on before you move to the next milestone.
   -  The window popup  on the mobile version does not appear as we have it on the figma design. 
   - The `X` icon to close the modal is not showing on the popup window.
   - The arrangement of the list of tools used to build the project are not properly layout as we have them on the figma design.
   - There's a small horizontal scroll bar at the bottom of the page.
 kindly attend to all these observations and fix them.


### You Design
your-mobile-windo-popup

### Screen Recording
[app-behave.webm](https://user-images.githubusercontent.com/26597432/233429136-05b6381b-2b61-49ec-9426-620b77482bdd.webm)

### Figma Design
temp-mobile-window-popup

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styling.css (hunk 1 above)
Comment (gisachris) [reply styling.css]: ### **As per my previous CR 🔍  i was requested ♻️  to make some changes like :**
**-make better design for mobile view to fit figma design🎨**
**-Remove the horizontal scroll bar 📜 to improve ux.**

as per my last commit 🟢  to the pull request i have fixed  🧑‍🔧  all the issues above. 🏅  
I hope to hear from you my CR 🤺  soon!.

**Review by franclobo — APPROVED** [review]
Hi @gisachris, 

# STATUS: APPROVED 😃 

git-merge

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- No linter errors ✔️ 
- Descriptive PR ✔️
- Correct Github flow ✔️ 
- Good UI ⭐ ⭐ ⭐  

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**